common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -157,9 +158,18 @@ public Random current() {
         DIRECT_MEMORY_LIMIT = maxDirectMemory;
         logger.debug("io.netty.maxDirectMemory: {} bytes", maxDirectMemory);
 
+        boolean tryAllocateUninitializedArray =
+                SystemPropertyUtil.getBoolean("io.netty.tryAllocateUninitializedArray", true);
+        TRY_ALLOCATE_UNINITIALIZED_ARRAY = tryAllocateUninitializedArray && hasUnsafe() && javaVersion() >= 9;
+        logger.debug("io.netty.tryAllocateUninitializedArray: {}", tryAllocateUninitializedArray);
```

Comment:
Not directly related with this PR, but weren't we used to prepend `-D` to system property names when logging them?